plugins/mcreator-localization/lang/texts.properties

```diff
@@ -848,4 +848,36 @@ elementgui.command.warning.empty_string=Command can't be empty string
 elementgui.fuel.burn_time=Fuel burn time (power): 
 elementgui.fuel.properties=Fuel properties
 modelement.painting=Painting
-modelement.painting.description=Painting is a picture that you can put on your walls using the Painting item
\ No newline at end of file
+modelement.painting.description=Painting is a picture that you can put on your walls using the Painting item
+elementgui.tool.stay_in_grid_when_crafting=Check to enable

```

Comment:
>elementgui.tool.stay_in_grid_when_crafting
>elementgui.tool.stay_in_grid_when_crafted
Please change one of them, otherwise, it won't work.

Reply:
Fixed!